### PR TITLE
adding the result bar section

### LINKED ISSUE #24: 1. C. Results bar
- [x] Create a div to display the results bar.
  - [x]  add span elements within this div to create the circles.
 

This user story is developed on the branch `resultsBarHTML/CSS`.
This branch is merged into the `master` branch after completion.

### LINKED ISSUE #25: 1. C. Results bar
- [x] style the container with the results bar.
  - [x] width
  - [x] position: bottom left of the webpage
  - [x] set the border-radius to 50% for the circles

This user story is developed on the branch `resultsBarHTML/CSS`.
This branch is merged into the `master` branch after completion.

### LINKED ISSUE #52: 1. A. Question Page
- [x] Create a HTML file for the questions's page.

This user story is developed on the branch `QuestionsPage`.
This branch is merged into the `master` branch after completion.

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I added the result bar section to the questions.html file as well styled it at index.css file

### CODE REVIEW

**Comment by Moamin-AbuEwaida** [comment]
this linting error of the styling file "lint:css" is driving me crazy, i fixed the error at my vscode and here it appears in a weird way :thought_balloon:

**Comment by JTLiberona** [comment]
Try fixing this errors first

 ( line : place )
 26:1   ✖  Expected empty line before comment                               comment-empty-line-before               
  43:1   ✖  Expected empty line before rule                                  rule-empty-line-before                  
 100:20  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after           
 102:16  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after           
 102:20  ✖  Unexpected whitespace before ";"                                 declaration-block-semicolon-space-before
 106:1   ✖  Expected no more than 1 empty line                               max-empty-lines

File: client/styles/index.css [hunk 1]
```diff
@@ -23,14 +23,15 @@ body {
   padding: 30px;
   border-radius: 10px;
 }
-
+/* i had to remove this line "vertical-align: middle;"
```
Comment (JTLiberona) [inline client/styles/index.css:26]: 26:1 heavy_multiplication_x Expected empty line before comment comment-empty-line-before

File: client/styles/index.css [hunk 2]
```diff
@@ -39,7 +40,6 @@ body {
   margin: 0 auto;
   display: block;
 }
-
 .button span {
```
Comment (JTLiberona) [inline client/styles/index.css:43]: 43:1 heavy_multiplication_x Expected empty line before rule rule-empty-line-before

File: client/styles/index.css (hunk 1 above)
Comment (Moamin-AbuEwaida) [inline client/styles/index.css:27]: I'll remove these empty spaces then

File: client/styles/index.css [hunk 3]
```diff
@@ -85,3 +85,21 @@ body {
...
+  border-radius: 10px;
+  background-color: white;
+}
+
+.result-indication {
+  height: 50px;
+  width: 50px;
+  background-color:#b8b8b8;
+  border-radius: 50%;
+  margin-right:10px ;
```
Comment (JTLiberona) [inline client/styles/index.css:102]: 100:20 heavy_multiplication_x Expected single space after ":" with a single-line declaration declaration-colon-space-after

File: questions.html [hunk 4]
```diff
@@ -4,13 +4,23 @@
...
   </head>
   <body>
-    <div id="questionContainer" class="question-container"></div>
+    <div id="questionContainer" class="question-container">
+      <p></p>
+    </div>
     <div id="answersContainer" class="answers-container"></div>
+    <div id="resultsBarContainer" class="resultsBar-container">
+      <div id="resultIndication" class="result-indication"></div>
+      <div id="resultIndication" class="result-indication"></div>
```
Comment (JTLiberona) [inline questions.html]: Error:   19:16  error  Duplicate ID "resultIndication"  no-dup-id

File: questions.html [hunk 5]
```diff
@@ -4,13 +4,23 @@
...
+      <div id="resultIndication" class="result-indication"></div>
+      <div id="resultIndication" class="result-indication"></div>
```
Comment (JTLiberona) [inline questions.html]: Error:   20:16  error  Duplicate ID "resultIndication"  no-dup-id

File: questions.html [hunk 6]
```diff
@@ -4,13 +4,23 @@
...
   <body>
-    <div id="questionContainer" class="question-container"></div>
+    <div id="questionContainer" class="question-container">
+      <p></p>
+    </div>
     <div id="answersContainer" class="answers-container"></div>
+    <div id="resultsBarContainer" class="resultsBar-container">
+      <div id="resultIndication" class="result-indication"></div>
+      <div id="resultIndication" class="result-indication"></div>
+      <div id="resultIndication" class="result-indication"></div>
```
Comment (JTLiberona) [inline questions.html]: Error:   21:16  error  Duplicate ID "resultIndication"  no-dup-id

File: questions.html [hunk 7]
```diff
@@ -4,13 +4,23 @@
...
+      <div id="resultIndication" class="result-indication"></div>
+      <div id="resultIndication" class="result-indication"></div>
```
Comment (JTLiberona) [inline questions.html]: Error:   22:16  error  Duplicate ID "resultIndication"  no-dup-id

**Review by JTLiberona — CHANGES_REQUESTED** [review]
You already fixed the CSS errors. Now the error is in the ID of the HTML file

questions.html
Error:   19:16  error  Duplicate ID "resultIndication"  no-dup-id
Error:   20:16  error  Duplicate ID "resultIndication"  no-dup-id
Error:   21:16  error  Duplicate ID "resultIndication"  no-dup-id
Error:   22:16  error  Duplicate ID "resultIndication"  no-dup-id

✖ 4 problems (4 errors, 0 warnings)

File: questions.html (hunk 7 above)
Comment (Moamin-AbuEwaida) [reply questions.html]: eeef :sweat_smile: 
done

**Comment by JTLiberona** [comment]
Ok! Now it is ready for approval ;)

**Review by JTLiberona — APPROVED** [review]

**Review by rayanejsilva — APPROVED** [review]

**Pull request merged**